Features added

Tkolby opened the pull request:
Added features and removed rend attributes from heads

whanley commented:
I added these changes manually to avoid the conflict. Still lots of bold face in the files, and many missing paragraph breaks.

Tkolby commented:
I will remove more bold face, but do believe that the titles I have bolded are representative of the newspaper headings. As for the paragraph breaks, I will sort through the files now to add additional paragraphs. My previous understanding was I was missing closing tags for the paragraphs I had already placed.

I no longer have push access for the blogposts and therefore cannot replace the feature image I have. Please see attached for the picture of the article.

Tyler
Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10


From: Will Hanley <notifications@github.com>
Sent: Monday, December 16, 2019 3:55:26 PM
To: dig-eg-gaz/content <content@noreply.github.com>
Cc: Tyler Kolby <tdk15b@my.fsu.edu>; Author <author@noreply.github.com>
Subject: Re: [dig-eg-gaz/content] Features added (#871)


I added these changes manually to avoid the conflict. Still lots of bold face in the files, and many missing paragraph breaks.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https:/github.com/dig-eg-gaz/content/pull/871?email_source=notifications&email_token=ANBHMERQ4HM2TMIUVRQQNWLQY7TL5A5CNFSM4J2TZIC2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEHAB6EQ*issuecomment-566238994__;Iw!!Epnw_ITfSMW4!919is7iakrwwhMoSOP9Kt5pzQtXxNgjhalEoaeuJx6Zw8bFO_nC1sNPOGj1qCMieiDg$>, or unsubscribe<https://urldefense.com/v3/__https:/github.com/notifications/unsubscribe-auth/ANBHMEVOAOR6FEXV3J3DWI3QY7TL5ANCNFSM4J2TZICQ__;!!Epnw_ITfSMW4!919is7iakrwwhMoSOP9Kt5pzQtXxNgjhalEoaeuJx6Zw8bFO_nC1sNPOGj1q5jVEQbM$>.
From: Will Hanley<mailto:notifications@github.com>
Sent: Monday, December 16, 2019 3:55 PM
Subject: Re: [dig-eg-gaz/content] Features added (#871)


I added these changes manually to avoid the conflict. Still lots of bold face in the files, and many missing paragraph breaks.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https:/github.com/dig-eg-gaz/content/pull/871?email_source=notifications&email_token=ANBHMERQ4HM2TMIUVRQQNWLQY7TL5A5CNFSM4J2TZIC2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEHAB6EQ*issuecomment-566238994__;Iw!!Epnw_ITfSMW4!919is7iakrwwhMoSOP9Kt5pzQtXxNgjhalEoaeuJx6Zw8bFO_nC1sNPOGj1qCMieiDg$>, or unsubscribe<https://urldefense.com/v3/__https:/github.com/notifications/unsubscribe-auth/ANBHMEVOAOR6FEXV3J3DWI3QY7TL5ANCNFSM4J2TZICQ__;!!Epnw_ITfSMW4!919is7iakrwwhMoSOP9Kt5pzQtXxNgjhalEoaeuJx6Zw8bFO_nC1sNPOGj1q5jVEQbM$>.
